components/content-blocks/Callout/CalloutEntry/styles.js

```diff
@@ -0,0 +1,206 @@
+import styled from "styled-components";
+import { MixedLink } from "@rubin-epo/epo-react-lib";
+import SharePopupComponent from "@/layout/SharePopup";
+import {
+  fluidScale,
+  containerRegular,
+  respond,
+  tokens,
+} from "@/styles/globalStyles";
+import { Section as CalloutSection } from "../styles";
+
+export const Section = styled(CalloutSection)`
+  position: relative;
+`;
+
+export const Inner = styled(MixedLink)`
+  ${containerRegular()}
+  position: ${(p) => p.$position};
+  display: grid;
+  gap: 40px;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    "image title"
+    "image subtitle"
+    "image text"
+    "image footer";
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 10px;
+  grid-column-gap: 30px;
+  min-height: 360px;
+  height: 100%;
+  padding-top: ${fluidScale("50px", "30px")};
+  padding-bottom: ${fluidScale("50px", "40px")};
+  align-content: start;
+  text-decoration: none;
+  transition: color 0.2s, background-color 0.2s;
+
+  // set the grid areas for various bits
+  .image {
+    grid-area: image;
+    overflow: hidden;
+    min-height: 300px;
+    opacity: 1;
+    transition: filter 0.2s, opacity 0.2s;
+
+    img {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+  .title {
+    grid-area: title;
+  }
+  .subtitle {
+    grid-area: subtitle;
+    margin-top: 0.727em;
+    font-size: 18px;
+    font-weight: 700;
+
+    ${respond(
+      `
+      font-size: 16px
+    `,
+      tokens.BREAK_MOBILE
+    )}
+  }
+  .text {
+    grid-area: text;
+  }
+  .footer {
+    display: grid;
+    grid-area: footer;
+    grid-template-columns: max-content 1fr;
+    place-items: end;
+    margin-top: 15px;
+
+    .c-buttonish {
+      &:hover {
+        outline: none;
+      }
+    }
+  }
+
+  ${respond(
+    `
+    grid-template-areas:
+      "image image"
+      "title title"
+      "subtitle subtitle"
+      "text text"
+      "footer footer";
```

Comment:
This may be the first CSS grid I ever made from scratch.  Middling confidence that I did it all correctly, though it seems to be working as expected on the frontend.

Reply:
looks good!